AxonDeepSeg/noise_simulation.py

```diff
@@ -1,63 +0,0 @@
-
```

Comment:
why was this file removed? what does that have to do with this PR?

Reply:
Nothing to do with this PR. Just a duplicate from a previous merge to master, I saw it now and wanted to resolve at the same time.

Reply:
in the future, please indicate at the beginning of the PR all the changes. Otherwise, it makes the review process confusing (i.e., looking for changes specific to the PR, but end up reviewing unrelated issues).